test/nsidc-icesat2/NSIDC-ICESAT2_Regression.ipynb

```diff
@@ -521,7 +521,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.11.10"
+   "version": "3.11.5"
```

Comment:
Does it matter that the Python version was downgraded in this notebook?

Reply:
That's just the metadata for who was running the notebook last, so it doesn't matter.